tests/comparisons/Migration/testCompositeConstraintsSnapshot.php

```diff
@@ -34,26 +34,31 @@ public function up()
             ])
             ->addIndex(
                 [
-                    'slug',
-                ],
-                ['unique' => true]
+                    'category_id',
+                ]
+            )
+            ->addIndex(
+                [
+                    'product_id',
+                ]
```

Comment:
Yeah me too.
Maybe it's due to the fact that the constraint from ``orders`` is a multi-columns one.
Out of curiosity, I'll do some research to see what this is about